markers/uncollect.py

```diff
@@ -68,23 +69,25 @@ def get_uncollect_function(marker_or_markdecorator):
 def uncollectif(item):
     """ Evaluates if an item should be uncollected
 
-    Tests markers against a supplied lambda from the marker object to determine
+    Tests markers against a supplied lambda from the markers object to determine
     if the item should be uncollected or not.
     """
 
     from cfme.utils.pytest_shortcuts import extract_fixtures_values
-    marker = item.get_marker('uncollectif')
-    if marker:
+    markers = item.get_marker('uncollectif')
+    if not markers:
+        return True
+    for mark in markers:
         from cfme.utils.log import logger
```

Comment:
Iwould probably move this out of the for loop, although probably nto much of perf influence, it seems a bit odd to me.